_includes/navigation.html

```diff
@@ -46,4 +47,14 @@
 		</ul>
 	
 	</div>
-</nav>
\ No newline at end of file
+</nav>
+<script>
```

Comment:
move the js script to javascript file not in html file.Keep things seaparate

Reply:
At least with respect to the function in this file, we can do it within CSS.

Reply:
Hi @sudhanshutech and @leecalcote. 
Does the changes address your comments.
Please let me know if you have any further requests